android/src/main/java/com/onesignal/flutter/OneSignalPlugin.java

```diff
@@ -337,6 +339,14 @@ public void onFailure(OneSignal.OSSMSUpdateError error) {
     });
   }
 
+  private void setLanguage(MethodCall call, final Result result) {
+    String language = call.argument("language");
+    if(language != null && language.length() == 0)
```

Comment:
Could you add a space after the if? Also, isn't there a string utils notNullorEmpty? I might be wrong

Reply:
I think this condition i referred from setExternalUserId looks for a condition where the string may be non-null but is empty. I which case it assigns null value to the string (possibly to push null to the endpoint instead of an empty string?)